app/pages/events/$id.mjs

```diff
@@ -1,9 +1,22 @@
+import { marked } from "marked"
+
 export default function ({ html, state = {} }) {
   let { store = {} } = state
   let { event } = store
+
   return html`
     <page-layout title=${event.title}>
-      <pre>${JSON.stringify(event, null, 2)}</pre>
+      <h3>${event.title}</h3>
+      ${event.description && `<p>${marked(event.description)}</p>` }
+
+      <h4>Thanks to our Sponsors ❤️</h4>
+      ${event.sponsors.length > 0 ? html`<event-sponsors></event-sponsors>` : null}
+      <event-talks-list></event-talks-list>
```

Comment:
Could we reuse `<list-talks>` element here? We may want to change it's style to `justify-content: start` on this page.

Reply:
I opted to create a new element since the data being sent to `index.html`, where `<list-talks>` is used, has all events in the data and filters the single event as opposed to the data for `events/$id` where it just receiving the single event from the data.  The choices that I see are:
1) Change the data being sent to `events/$id` receives all the events to filter.
2) Reuse `<list-talks>` and add conditions to check the store for events or a single event.
3) Something else entirely that I'm not thinking of and you may be able to offer guidance on. 

I feel option 1 (or 3) seems like the best bet and just have a bit of data over fetching.  I'm open to any of these but what do you think?

Reply:
I made  a change to the api in the end.

Reply:
Oh right. Yea I always found it odd in this system that elements are implicitly tied to a certain data contract that the api must fulfill. There's no way to know that... sigh...

Anyway, what you did looks good! Thank you.